database/stats.go

```diff
@@ -6,16 +6,20 @@ import (
 
 const statsQuery = `
 SELECT COALESCE(SUM(serviceFee + onchainFee), 0), COALESCE(SUM(expectedAmount), 0), COUNT(*)
-FROM (SELECT serviceFee, onchainFee, expectedAmount, isAuto, createdAt, false isChain
+FROM (SELECT serviceFee, onchainFee, expectedAmount, isAuto, createdAt, false isChain, entityId
       FROM swaps
       UNION ALL
-      SELECT serviceFee, onchainFee + (routingFeeMsat / 1000), expectedAmount, isAuto, createdAt, false isChain
+      SELECT serviceFee, onchainFee + (routingFeeMsat / 1000), expectedAmount, isAuto, createdAt, false isChain, entityId
       FROM reverseSwaps
-      ) stats
+      UNION ALL
+      SELECT serviceFee, onchainFee, data.amount, isAuto, createdAt, true isChain, entityId
+      FROM chainSwaps
+      JOIN chainSwapsData data ON chainSwaps.id =data.id AND data.currency = chainSwaps.fromCurrency
+      ) stats 
 `
 
 func (database *Database) QueryStats(args SwapQuery, chainOnly bool) (*boltzrpc.SwapStats, error) {
```

Comment:
`chainOnly` implies that setting it to false means that all swap types are included. What it actually does is either chain or submarine and reverse

Reply:
what about a `swapTypes []boltz.SwapType` as an argument?

Reply:
That'd be best imho